debezium-core/src/test/java/io/debezium/pipeline/source/snapshot/incremental/SignalBasedSnapshotChangeEventSourceTest.java

```diff
@@ -1,299 +0,0 @@
-/*
```

Comment:
Are you sure the test hav nothing in common? IMHO it might make sense to have either 3 test classes or two test classes with a common parent with shared tests.